prueba/common/client-app.js

```diff
@@ -0,0 +1,58 @@
+/// <reference path="../../node_modules/rfc6902/rfc6902.d.ts" />
+var rfc6902 = require("rfc6902");
+var utils = require("./utils");
+exports.defaultConfiguration = {
+    rootNode: null,
+    socketEventName: "board",
+    throttlingInterval: 1000
+};
+function start(boardVM, config) {
+    var app = new App(boardVM, config);
+    app.start();
+    return app;
+}
+exports.start = start;
```

Comment:
If I am not wrong, you can avoid this line prepending original function declaration with `export`, for example:

```
export function start(boardVM, config) {
    var app = new App(boardVM, config);
```